modules/arima/filterfiveend/meta.yml

```
@@ -0,0 +1,54 @@
+name: arima_filterfiveend
+description: Filter BAM files using Arima Filter five end
+keywords:
+  - filter
+  - filterfiveend
+  - samtools
+tools:
+  - arima:
+      description: A wrapper tool around Arima Genomics's filter_five_end.pl to filter BAN files.
+      homepage: https://arimagenomics.com
+      documentation: https://usermanual.wiki/Document/ArimaMappingUserGuideA160156v01.424413804/view
```

Do you know if this file was uploaded by Arima themselves ? If you're not sure, better to use the version that is on their GitHub
```suggestion
      documentation: https://github.com/ArimaGenomics/mapping_pipeline/blob/master/Arima_Mapping_UserGuide_A160156_v02.pdf
```